jupyter_lidar_inspection.ipynb

```diff
@@ -0,0 +1,918 @@
+    "smaller_region_dict.update(updates)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d7839235-f132-49bf-bee0-fc2490f07f09",
+   "metadata": {},
+   "source": [
+    "Let's update momentarily our region so we can visualize a smaller portion of it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe7dc980",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n=int(smaller_region_dict['n'])\n",
+    "s=int(smaller_region_dict['s'])\n",
+    "w=int(smaller_region_dict['w'])\n",
+    "e=int(smaller_region_dict['e'])\n",
+    "univar_stat=gs.parse_command('r.univar',map='point_cloud_1m', flags='g')\n",
+    "z_min=float(univar_stat['min'])\n",
+    "z_max=float(univar_stat['max'])\n",
+    "\n",
+    "gs.run_command(\"g.region\",\n",
+    "               save='smaller_region',\n",
+    "               n=n,\n",
+    "               s=s,\n",
+    "               w=w,\n",
+    "               e=e,\n",
+    "               flags=\"d\")\n",
+    "gs.run_command(\"g.region\",region='smaller_region')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b5b70bf8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gs.run_command(\"g.region\",\n",
+    "               save='smaller_region',\n",
+    "               n=int(smaller_region_dict['n']),\n",
+    "               s=int(smaller_region_dict['s']),\n",
+    "               w=int(smaller_region_dict['w']),\n",
+    "               e=int(smaller_region_dict['e']),\n",
+    "               flags=\"d\")\n",
+    "gs.run_command(\"g.region\",region='smaller_region')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e356497-1662-443c-9de4-689e139a8c10",
+   "metadata": {},
+   "source": [
+    "## 3D Data Visualization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "755e62d4",
+   "metadata": {},
+   "source": [
+    "Let's do a 3D check of the region: tiledb will manage that our computer survives"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d4815a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = os.path.expanduser(\"/your/path/point_cloud.laz\")\n",
+    "array_name = os.path.expanduser(\"/your/path/tiledb_pc\")\n",
+    "filename = \"/your/path/point_cloud.laz\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "114bc70b-a904-4c2f-8331-e585eda9b8e4",
+   "metadata": {},
+   "source": [
+    "Let's create a PDAL pipeline to import our LAZ into a tiledb database."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d84f2b86",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "pipeline = pdal.Reader.las(filename=data).pipeline()\n",
+    "pipeline |= pdal.Writer.tiledb(filename=array_name, x_tile_size=200, y_tile_size=200, z_tile_size=100)\n",
+    "pipeline.execute() # once we create it, we can just load it next time\n",
+    "# The output states the number of points"
```

Comment:
Both here and below, please consider adding some minimal explanations and links to the other software packages/functions for reference